environment.yml

```diff
@@ -17,11 +17,15 @@ dependencies:
 - pulp
 - paramtools
 - pypandoc
+- openpyxl
+- lxml_html_clean
+- plotly
+- kaleido
 - selenium
 - python-chromedriver-binary
 - tabulate
 - pre-commit
-- taxcalc>=3.5.0
+- taxcalc>=5.3.0
```

Comment:
I've noticed an issue trying to install TC 5.3 from conda.  Try to pip install instead (as done in current master branch).